modules/reference/pages/cluster-properties.adoc

```diff
@@ -489,6 +489,30 @@ When the disk usage of a node exceeds this threshold, it triggers Redpanda to mo
 
 ---
 
+=== write_caching
+
+The write caching mode to apply to a cluster. Write caching acknowledges a message as soon as it is received and acknowledged on a majority of brokers, without waiting for it to fsync to disk. With `acks=all`, this provides lower latency while still ensuring that a majority of brokers acknowledge the write. Fsyncs follow `raft_replica_max_pending_flush_bytes` and `raft_replica_max_flush_delay_ms`, whichever is reached first. 
+
+Valid modes:
+
+* `off`
+* `on`
+* `disabled`: This takes precedence over topic overrides and disables write caching for the entire cluster.
+
+This can be overridden with the topic property xref:topic-properties.adoc#writecaching[`write.caching`].
```

Comment:
The link xref:topic-properties.adoc#writecaching[`write.caching`] refers the reader to this same property--self referential. I'd remove this link.